components/brave_rewards/resources/android_page/components/pageWallet.tsx

```diff
@@ -133,17 +133,19 @@ class PageWallet extends React.Component<Props, State> {
               actions={[]}
               compact={true}
               isMobile={true}
-              showCopy={true}
+              showCopy={false}
```

Comment:
I think this one should be true if `onlyAnonWallet` is false. This is at least how we have it on desktop. So only false for `onlyAnonWallet` case

Reply:
Fixed, good catch!